__tests__/app.test.js

```diff
@@ -122,52 +122,68 @@ describe("GET /api/articles", () => {
 })
 
 
-describe("GET /api/articles/:article_id/comments", () => {
-  test("200: Responds with an array of comments for a given article_id, each comment with correct properties", () => {
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    const newComment = {
+      username: 'butter_bridge',
+      body: 'comment',
+    }
+
     return request(app)
-      .get('/api/articles/1/comments')
-      .expect(200)
-      .then(({ body: { comments } }) => {
-        expect(comments).toHaveLength(11)
-        comments.forEach((comment) => {
-          expect(comment).toMatchObject(
-            {
-              comment_id: expect.any(Number),
-              votes: expect.any(Number),
-              created_at: expect.any(String),
-              author: expect.any(String),
-              body: expect.any(String),
-              article_id: expect.any(Number),
-            }
-          )
+       .post('/api/articles/1/comments')
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          author: 'butter_bridge',
+          body: 'comment',
+          article_id: 1,
+          votes: 0,
+          created_at: expect.any(String),
```

Comment:
Nice test! Are we able to be a little more specific on the comment_id? If yes or no, why?